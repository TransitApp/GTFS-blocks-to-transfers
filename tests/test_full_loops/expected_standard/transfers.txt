from_trip_id,to_trip_id,transfer_type
trip_1,trip_2,4
trip_2,trip_3,4
