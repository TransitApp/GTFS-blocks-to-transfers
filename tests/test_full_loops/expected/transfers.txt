from_trip_id,to_trip_id,transfer_type
trip_1,trip_2,5
trip_2,trip_3,5
